website/static/js/onboarder.js

```diff
@@ -596,6 +597,10 @@ function OBUploaderViewModel(params) {
     };
     self.dropzone = new Dropzone(self.selector, dropzoneOpts);
 
+    if($osf.isIE()){
```

Comment:
Any workarounds like this should be accompanied by a code comment explaining why it's necessary.